.devcontainer/conf/php/Dockerfile

```
@@ -0,0 +1,58 @@
+FROM php:8.2-fpm-alpine
+
+RUN apk update && \
+    # Install tools
+    apk add nodejs-current npm mysql-client curl patch gettext perl git \
+    zsh sudo vim htop && \
+    # Install Python, pip and Python modules required by "plugin-release" script.
+    apk add python3-dev py3-pip && \
+    apk add libffi-dev libxml2-dev libxslt-dev && pip install lxml && \
+    apk add git && pip install gitpython && \
+    pip install PyGithub && \
+    pip install termcolor && \
+    # Install PHP extensions dependencies
+    apk add --virtual .build-deps openldap-dev linux-headers $PHPIZE_DEPS && \
+    # Install PHP extensions
+    ## BZ2 extension
+    apk add bzip2-dev && \
+    docker-php-ext-install bz2 && \
+    ## Exif extension
+    docker-php-ext-install exif && \
+    ## GD extension
+    apk add freetype-dev libjpeg-turbo-dev libpng-dev libwebp-dev && \
+    docker-php-ext-configure gd --with-freetype --with-jpeg --with-webp && \
+    docker-php-ext-install gd && \
+    ## Intl extension
+    apk add icu-dev icu-data-full && \
+    docker-php-ext-install intl && \
+    ## LDAP extension
+    apk add libldap && \
+    docker-php-ext-install ldap && \
+    ## Mysqli extension
+    docker-php-ext-install mysqli && \
+    ## Opcache extension
+    docker-php-ext-install opcache && \
+    ## Zip extension
+    apk add libzip-dev && \
+    docker-php-ext-configure zip && docker-php-ext-install zip && \
+    ## Xdebug extension
+    pecl install xdebug && docker-php-ext-enable xdebug && \
+    # clean cache and dependencies
+    rm -rf /var/cache/apk/* && \
+    apk del -f .build-deps
```

I do not know if `apk del` write files into `/var/cache/apk/`.

```suggestion
    # Clean cache and dependencies
    apk del -f .build-deps && \
    rm -rf /var/cache/apk/*
```